### ISSUE TITLE
set up Gulp sass

### ISSUE THREAD

**Issue by hiplunatic** [pokemon-club/admin#6]
set up sass, and get the gulp task running

**Comment by IN3D** [comment]
@hiplunatic, I've implemented something similar on another project, [this](https://github.com/MekManager/mekmanager_client/blob/master/gulpfile.js#L95) might be a good reference.

**Comment by IN3D** [comment]
Also, question. Are we for sure on Sass, or might we want stylus?

**Comment by hiplunatic** [comment]
I am used to Sass, but I don't have any objections to using stylus.  which would you prefer?

**Comment by IN3D** [comment]
I have more experience with Sass too. I just wanted to check. And now that I think of it, Materialize is written in Sass, so we'd get better interop with our front end framework by using sass.

**Comment by IN3D** [comment]
Note: Impending code dump.

@hiplunatic, as I mentioned. I wanted to try and see if I could get `.sass` and `.scss` files to interop. I have code samples for how to implement this in gulp.

The Sass, and SCSS files work together perfectly, so long as both a regular call to sass, and one with a call to sass with `{indentationSyntax: true}` are provided.

I am personally partial to the white space style, but I understand a lot of people find it weird.

Thoughts?

---

**gulpfile.js**

``` js
var gulp = ('gulp')
    sass = ('gulp-sass');

gulp.task('sass', function() {
  gulp.src(['./sass/*.sass', './sass/*.scss'])
      .pipe(sass())
      .pipe(sass({indentationSyntax: true})
      .on('error', function(e) { this.emit('end') })
      .pipe(gulp.dest('./css/'));
});
```

---

**sass/test.sass**

``` sass
@import 'wat'

$font-stack: Helvetica, sans-serif
$primary-color: #333

body
  font: 100% $font-stack
  color: $primary-color
```

---

**sass/_wat.scss**

``` scss
div {
  background-color: #fff;
  font-size: 12px;
  p {
    color: #333;
  }
}
```

**Comment by IN3D** [comment]
Closed in #6.
